packages/cycle-scripts-one-fits-all/template/src/pages/page2.tsx

```diff
@@ -0,0 +1,76 @@
+import xs, { Stream } from 'xstream';
+import { VNode, DOMSource } from '@cycle/dom';
+import { StateSource } from 'cycle-onionify';
+
+import { DriverSources, DriverSinks } from '../drivers';
+
+export interface State {
+    count : number;
+}
+export type Reducer = (prev : State) => State | undefined;
+export type Sources = DriverSources & { onion : StateSource<State> };
+export type Sinks = DriverSinks & { onion : Stream<Reducer> };
+
+export function Page2(sources : Sources): Sinks {
```

Comment:
I think it's better to show state persistance with two completely distinct components